apps/lumberjack-app/project.json

```diff
@@ -84,6 +84,7 @@
     },
     "use-lumberjack-bundle": {
       "executor": "nx:run-commands",
+      "dependsOn": ["^build"],
```

Comment:
praise: Nice find!

Reply:
Thank you, yeah the result was much more clear!